### DISCUSSION TITLE
Support me

### DISCUSSION THREAD

**Discussion by superdev223** [community/community#58113] [Accessibility]
### Select Topic Area

Question

### Body

I have done googling how can I do git commit and it said there are few ways.

Using VSCode, Git desktop app and others . Which one is better?

**Comment by HappyMicky0317** [comment]
The best way to do a git commit depends on your personal preference and workflow. Here are some of the most popular options:

1. Using the Git command line: This is the most basic way to do a git commit. It requires you to use the command line interface to execute git commands. This option is great if you prefer working with the command line and want more control over your git workflow.

2. Using a Git GUI: Git GUIs like GitHub Desktop and GitKraken provide a graphical user interface for executing git commands. This option is great if you prefer a visual interface and want to simplify your git workflow.

3. Using an IDE with built-in Git support: IDEs like VSCode, IntelliJ, and Eclipse have built-in Git support that allows you to execute git commands from within the IDE. This option is great if you prefer an all-in-one development environment and want to streamline your workflow.

Ultimately, the best way to do a git commit is the one that works best for you and your workflow. You may want to try out a few different options to see which one fits your needs the best.

**Reply by superdev223** [reply]
Thanks

**Comment by ghostinhershell** [comment]
@superdev223  & @HappyDream0317, 

GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
